src/field/arena.ts

```diff
@@ -168,19 +168,14 @@ export class Arena {
 
       ret = getPokemonSpecies(species!);
 
-      if (ret.subLegendary || ret.legendary || ret.mythical) {
+      if (ret.subLegendary || ret.legendary || ret.mythical || ret) {
```

Comment:
```suggestion
      if (ret.subLegendary || ret.legendary || ret.mythical) {
```
Putting `ret` at the end there makes the other 3 redundant and makes it so _every_ species regens before wave 55